prepare-build.sh

```diff
@@ -17,11 +17,11 @@
 declare -a devices=("anthias" "bass" "beluga" "catfish" "dory" "firefish" "harmony" "koi" "inharmony" "lenok" "minnow" "mooneye" "narwhal" "nemo" "pike" "qemux86" "ray" "smelt" "sparrow" "sparrow-mainline" "sprat" "sturgeon" "sawfish" "skipjack" "swift" "tetra" "triggerfish" "wren")
 
 declare -a layers=(
-    "src/oe-core                   https://github.com/openembedded/openembedded-core.git kirkstone"
-    "src/oe-core/bitbake           https://github.com/openembedded/bitbake.git           2.0"
-    "src/meta-openembedded         https://github.com/openembedded/meta-openembedded.git kirkstone"
-    "src/meta-qt5                  https://github.com/meta-qt5/meta-qt5                  kirkstone"
-    "src/meta-smartphone           https://github.com/shr-distribution/meta-smartphone   kirkstone 038458fd1fe3d946ce084865adb09238c12eaf90"
+    "src/oe-core                   https://github.com/openembedded/openembedded-core.git mickledore"
+    "src/oe-core/bitbake           https://github.com/openembedded/bitbake.git           2.4"
+    "src/meta-openembedded         https://github.com/openembedded/meta-openembedded.git c5f330bc9ae72989b8f880aa15e738a3c8fce4e7"
```

Comment:
This line is not correct. You need to supply a branch simply a hash won't work:
```
"src/meta-openembedded         https://github.com/openembedded/meta-openembedded.git master c5f330bc9ae72989b8f880aa15e738a3c8fce4e7"
```

Reply:
Fixed!